invoices/invoiceregistry.go

```diff
@@ -28,41 +28,6 @@ var (
 	DebugHash = DebugPre.Hash()
 )
 
-// HtlcCancelReason defines reasons for which htlcs can be canceled.
-type HtlcCancelReason uint8
-
-const (
```

Comment:
It may still be useful to log this  information internally for debugging purposes (dev using two nodes on simnet for example).

Reply:
We do, see the `debugLog` statements in `NotifyExitHopHtlc`